cypress/support/cypress-commands/lifecycle.js

```diff
@@ -408,21 +398,19 @@ Cypress.Commands.add('setAppState', (state, appId) => {
 
     // Set state to suspended
     case CONSTANTS.LIFECYCLE_STATES.SUSPENDED:
-      // If current app state is not suspended or inactive,  set app state to inactive first to comply with allowed transitions
-      if (currentAppState.state != CONSTANTS.LIFECYCLE_STATES.SUSPENDED) {
-        if (currentAppState.state != CONSTANTS.LIFECYCLE_STATES.INACTIVE) {
-          cy.setAppState(CONSTANTS.LIFECYCLE_STATES.INACTIVE, appId);
-        }
-        // Send lifecycle.suspend API call to 3rd party app
-        cy.invokeLifecycleApi(appId, CONSTANTS.LIFECYCLE_APIS.SUSPEND, {}).then((response) => {
-          if (response) {
-            cy.log(CONSTANTS.APP_RESPONSE + JSON.stringify(response));
-          }
-          appObject.setAppObjectState(state);
-          cy.lifecycleSchemaChecks(response, state);
-          // TODO: Checks for platform support
-        });
+      // If current app state is not inactive,  set app state to inactive first to comply with allowed transitions
+      if (currentAppState.state != CONSTANTS.LIFECYCLE_STATES.INACTIVE) {
+        cy.setAppState(CONSTANTS.LIFECYCLE_STATES.INACTIVE, appId);
       }
+      // Send lifecycle.suspend API call to 3rd party app
+      cy.invokeLifecycleApi(appId, CONSTANTS.LIFECYCLE_APIS.SUSPEND, {}).then((response) => {
+        if (response) {
+          cy.log(CONSTANTS.APP_RESPONSE + JSON.stringify(response));
+        }
+        appObject.setAppObjectState(state);
+        cy.lifecycleSchemaChecks(response, state);
+        // TODO: Checks for platform support
+      });
```

Comment:
No need to make any change to background as the api itself is not present in open rpc